client/blocks/upgrade-nudge-expanded/style.scss

```diff
@@ -44,13 +44,21 @@
 	}
 
 	.upgrade-nudge-expanded__title-plan-icon {
-		width: 54px;
-		height: 54px;
+		width: 55px;
```

Comment:
We should update the nudge to use a plan icon component instead:
https://github.com/Automattic/wp-calypso/tree/master/client/components/plans/plan-icon
e.g:
```javascript
<PlanIcon plan={ PLAN_BUSINESS } />
```